thirdparty/lws/mbedtls_wrapper/include/platform/ssl_port.h

```diff
@@ -24,7 +24,12 @@
 #include "esp_log.h"
 */
 #include "string.h"
+
+#if defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
I guess you could merge this one in its parent commit, which undid the fix. The parent commit also has some Godot-specific changes, so you can just mark those as Godot-specific until they're actually replaced by an upstream release.